static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -321,6 +324,24 @@ const SPECIAL_FIELDS: SpecialFields = {
       return <Container>{emptyValue}</Container>;
     },
   },
+  'count_unique(user)': {
+    sortField: 'count_unique(user)',
+    renderFunc: data => {
+      const count = data.count_unique_user;
+      if (count && typeof count === 'number') {
```

Comment:
Is a count of 0 (zero) unique users still valid? Otherwise, we'd be showing `n/a` for count of 0 unique users.

Reply:
Good catch @dashed

Reply:
Yep, added a check that wasn't needed. To Tony's question, I think if we receive 0 we should show 0, if we received undefined or something that isn't a number we can show 'n/a'.